test_full_review.py

```
@@ -0,0 +1,73 @@
+        data[i] *= 2
+
+    sum_data = 0
+    for item in data:
+        sum_data += item
+    
+    return sum_data / len(data)
+
+# 6.การรักษาความปลอดภัย (Security)
+def get_user_data(user_id):
+    query = "SELECT * FROM users WHERE id = " + user_id
+    execute_database_query(query)
```

Ruff (0.8.2)

Undefined name `execute_database_query`

(F821)